env.example

```diff
@@ -1,5 +1,4 @@
 # API server
 SERVER_PROTOCOL=http://
 SERVER_DOMAIN=localhost
-SERVER_PORT=7080
```

Comment:
@tschaffter Based on the line of code [here](https://github.com/Sage-Bionetworks/schematic/pull/1202/files#diff-a0f6c3ca194fef3726d45adc2ac51fb28aa30a82b8d7b7ab0f80a50496dc31e9:~:text=default%20to%2080-,USE_LISTEN_PORT,-%3D%24%7BLISTEN_PORT%3A%2D), the bash script is looking for env variable `USE_LISTEN_PORT`. Do you think if we should let users define `USE_LISTEN_PORT` in their own environment file? And we could also change `docker-compose.yml` to something like: 
```
    ports: 
      - "${LISTEN_PORT}:80"
      - "443:443"
  ```